nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/resources/es-audit-pageprovider-test-contrib.xml

```diff
@@ -94,6 +94,40 @@
       <pageSize>5</pageSize>
     </genericPageProvider>
 
+    <genericPageProvider name="CUSTOM_HISTORY_VIEW"
```

Comment:
I thought the fix needs to be done on EVENTS_VIEW, you are only changing unit test  how it can help to fix the NXP?

Reply:
this PR is related to the another jsf PR: nuxeo/nuxeo-jsf-ui#33